examples/manifold/plot_compare_methods.py

```diff
@@ -48,72 +49,33 @@
 ax.scatter(X[:, 0], X[:, 1], X[:, 2], c=color, cmap=plt.cm.Spectral)
 ax.view_init(4, -72)
 
-methods = ['standard', 'ltsa', 'hessian', 'modified']
-labels = ['LLE', 'LTSA', 'Hessian LLE', 'Modified LLE']
-
-for i, method in enumerate(methods):
+# Set-up manifold methods
+LLE = partial(manifold.LocallyLinearEmbedding,
+              n_neighbors, n_components, eigen_solver='auto')
+
+methods = dict()
+methods['LLE'] = LLE(method='standard')
```

Comment:
This dict will only be iterated in deterministic order in cython >= 3.6. I'm not certain if we are ready to rely upon that yet.

Reply:
Right, do you think the collections.OrderedDict I added is fine here?

Reply:
Yes, that seems a reasonable solution for now!